QUEST_LV_0100.tsv

```diff
@@ -6624,8 +6624,8 @@ QUEST_LV_0100_20150406_006623	Defeat Unknocker and get %s
 QUEST_LV_0100_20150414_006624	They are already gone from this world. {nl}They are repeating what happened back then because of their anger for Gaigalas. 
 QUEST_LV_0100_20150414_006625	This cable car does work. {nl}But the monsters around make it such a hassle to maintain it. 
 QUEST_LV_0100_20150414_006626	Can you take care of it? 
-QUEST_LV_0100_20150414_006627	Better use the cable car rather than going throught the monsters. 
-QUEST_LV_0100_20150414_006628	I thinks monsters are crawling out of the mysterious barrier in Nuotori Cliff. {nl}Can you defeat that barrier? 
+QUEST_LV_0100_20150414_006627	$Isn't it better to use the cable car rather than getting injured passing through the monsters?
```

Comment:
Why did you change it to a question? It's not a question, it's advice.
Also, I am pretty sure I have commented on this line before in another, merged PR.

"It's better to use the cable car rather than fighting your way past the monsters."